assets/css/tw.css

```diff
@@ -183,6 +183,12 @@
 .content-aside ol > li + li {
   @apply mt-3;
 }
+.spl-input {
+  @apply w-full rounded border border-s-4 py-2 pl-4 pr-7 font-sans outline-none ring-2 ring-transparent focus:ring-yellow;
+}
```

Comment:
At one point I wrote a tool to go through our CSS and see how many times a set of classes were used together: https://github.com/carlmjohnson/classy

Here's the output from master:

<details>

```
18  "fill-current h-4 w-4"
12  "absolute inline-flex inset-y-0 items-center justify-center leading-none left-0 pointer-events-none text-s-4 w-10 z-10"
11  "max-w-prose mx-auto w-full"
 8  "h-4 mr-2 w-4"
 7  "bg-white px-6 py-8"
 7  "has-colored-svg has-text-white icon"
 7  "flex-grow relative"
 6  "max-w-screen-lg mx-auto"
 6  "bg-red font-semibold inline-block px-4 py-3 rounded-br sm:text-lg text-white tracking-wider uppercase"
 6  "control is-expanded"
 6  "appearance-none bg-white block border-2 border-s-4 focus:ring-2 outline-none placeholder-opacity-100 placeholder-s-4 px-4 py-2 ring-red-7 rounded text-s-8 tracking-wider w-full"
 6  "fill-current h-6 w-6"
 6  "items-stretch mt-8 mx-auto sm:flex sm:max-w-2xl"
 5  "bg-white md:pb-6 pb-2 pt-12 px-4"
 5  "block font-extrabold mt-4"
 5  "bg-white p-6"
 5  "image is-64x64"
 5  "max-w-6xl mx-auto px-5"
 5  "fill-current h-8 inline-block w-48"
 4  "absolute bg-s-4 inset-0 opacity-75"
 4  "border border-red font-raleway font-semibold leading-tight rounded shadow"
 4  "border-b-4 border-black font-extrabold font-sans leading-none pb-2 text-xl tracking-wider uppercase"
 4  "flex flex-0 items-center mr-2"
 4  "basis-0 flex gap-2 items-start justify-start"
 4  "flex items-center"
 4  "absolute backdrop-blur-sm backdrop-filter inset-0"
 4  "bg-yellow-4 border-2 border-yellow-5 focus:ring-2 font-bold outline-none px-8 py-2 ring-yellow-6 rounded text-black tracking-widest uppercase w-full"
 4  "flex-grow hover:underline text-s-6"
 4  "mt-1 text-left"
 4  "appearance-none bg-white block border-2 border-s-4 focus:ring-2 outline-none pl-10 placeholder-opacity-100 placeholder-s-4 py-2 ring-yellow-6 rounded text-s-8 tracking-wider w-full"
 4  "mt-2 text-left"
 4  "hidden sm:block"
 4  "mb-4 md:mt-4 mr-3 mt-10 text-right text-s-9"
 4  "mt-4 sm:ml-1 sm:mt-0"
 4  "max-w-screen-lg mx-auto w-full"
 4  "max-w-screen-md mx-auto"
 4  "break-all text-blue"
 3  "flex gap-2 items-baseline"
 3  "flex flex-row items-center md:my-0 mr-10 my-2"
 3  "flex justify-center mt-12"
 3  "fixed inset-0 z-20"
 3  "is-monochrome is-rounded"
 3  "fill-current h-4 inline-block w-4"
 3  "appearance-none border-s-1 checked:bg-black h-6 mr-4 order-first rounded-full shadow-[0_0_0_3px_rgba(255,255,255,1)] w-6"
 3  "leading-tight min-w-full mt-4 sm:text-base sm:w-0 space-y-4 text-sm"
 3  "duration-500 hover:text-yellow text-navy transition-colors underline"
 3  "duration-500 hover:text-yellow text-navy transition-colors"
 3  "mt-12 text-center"
 3  "duration-500 hover:text-yellow text-blue transition-colors"
 3  "drop-shadow-outline fill-current filter h-8 hover:text-yellow lg:h-10 text-white"
 3  "is-child tile"
 3  "font-bold font-sans leading-none md:text-6xl sm:text-4xl text-3xl text-center tracking-wide uppercase"
 3  "bg-white font-serif pb-12 pt-6 px-4"
 3  "control has-text-left"
 3  "control has-icons-left has-icons-right is-expanded"
 3  "accent-black border border-black h-4 hover:accent-yellow-4 mr-3 p-1 rounded w-4"
 3  "is-4 subtitle"
 3  "columns is-centered is-multiline"
 3  "font-bold font-sans uppercase"
 3  "column is-three-fifths"
 3  "font-bold text-lg uppercase"
 3  "absolute flex inset-1 overflow-y-auto"
 3  "absolute bottom-0 inset-x-0 sm:hidden"
 3  "font-normal font-sans leading-tight my-3 text-base text-byline tracking-normal"
 3  "column is-3"
 3  "max-w-screen-lg mb-8 mt-4 mx-auto w-full"
 3  "font-sans mb-4 sm:mb-0 sm:mt-[95px] sm:shape-outside sm:w-auto w-full"
 3  "border-b-2 border-black tracking-wide whitespace-nowrap"
 3  "icon is-right validator-icon"
 3  "icon is-large is-left"
 3  "group-hover:underline text-s-7"
 3  "max-w-screen-lg mx-auto py-12"
 3  "has-text-weight-semibold is-uppercase"
 3  "button has-text-weight-semibold is-danger"
 3  "absolute bg-gradient-to-t from-white inset-0 sm:hidden"
 3  "flex gap-1 items-center"
 3  "bg-red-6 hover:bg-red-5 inline-block px-4 py-3 rounded text-white"
```

</details>